custom/icds_reports/tasks.py

```diff
@@ -1199,6 +1199,46 @@ def build_incentive_report(agg_date=None):
         agg_date = current_month - relativedelta(months=1)
     for state in state_ids:
         AWWIncentiveReport.aggregate(state, agg_date)
+
+    awc_performance_rows_list = list(AWWIncentiveReport.objects.values_list('awc_id', 'is_launched',
```

Comment:
This is going to be slower and slower by time Because this is comparing data from infinite past. 
It might be better to just compare for the latest month. Whats your thoughts @calellowitz

Reply:
Also, There could be lacs of rows and that would increase with the frequency of lacs.